tests/structuredheadings/formatcombo.js

```diff
@@ -33,23 +38,50 @@
       });
     },
 
-    "if previous heading is autonumber, the new heading is autonumbered": function () {
+    "increments autonumbering levels and preset": function () {
```

Comment:
This should be, more accurately:  

"when creating a new autonumbered heading of the next level, the level is incremented and the  numbering scheme matches the expected class for the next level"